Makefile

```diff
@@ -19,11 +19,27 @@ tmp/dev_image_id:
 format: prepare
 	${DOCKRUN} bash ./scripts/format.sh
 
-check: prepare
+check: prepare format
 	${DOCKRUN} bash ./scripts/check.sh
 
-todo: prepare
+todo:
 	${DOCKRUN} bash ./scripts/todo.sh
 
-test: prepare format check
+test: check db_seed
 	${DOCKRUN} bash ./scripts/test.sh
+
+db_start: db_stop
+	@docker run --name unit-testing-mysql-db -e MYSQL_ALLOW_EMPTY_PASSWORD=yes -p 3306:3306 -d mysql:5.6
+	@docker run --name unit-testing-mongo-db -p 27015-27017:27015-27017 -d mongo:4.2.0
+
+db_seed: db_start
+	@docker cp apna_school.sql unit-testing-mysql-db:apna_school.sql
+	@docker cp apna_school.mongo unit-testing-mongo-db:apna_school.mongo
+	@echo "Executing databases...wait for 30 seconds"
+	@sleep 30
```

Comment:
Can we verify that if we reduce tis to 20 or 15... does it work in that case?

Reply:
Because this adds up 30 seconds to the build time... we want to keep it as minimal as we can.